Fixed some typos

coding-pivo opened the pull request:
I just fixed a few typos within comments. I would like to contribute to the base project instead of using my own fork in the future. So this is just a check if this project is still maintained and willing to merge pull requests as the last commit was long time ago :)

Best regards

sevoku commented:
Oh wow, thanks for the PR! LGTM at the frist sight, but I need to review the changes thorougly first. The last time I've been coding VB is many years ago.

coding-pivo commented:
I'm glad that the project is still alive. Mostly I fixed some typos and updated the project for the latest Doxygen version. I will start to develop in branches and not "spamming" the master so that the pull request is not growing every day ;)

coding-pivo commented:
I just thought about the merging strategy and probably it is not a good idea to start a pull request from forked master to original master. In the future I will do the development in branches and start a pull request from forked dev branch to original master once the work is done. What do you think?
